scipy/sparse/linalg/_dsolve/_superlumodule.c

```diff
@@ -263,6 +264,173 @@ static PyObject *Py_gstrf(PyObject * self, PyObject * args,
     return NULL;
 }
 
+static PyObject *Py_gstrs(PyObject * self, PyObject * args,
+                          PyObject * keywds)
+{
+    /* compressed sparse row matrix L */
+    int L_N = 0, L_nnz = 0;
+    PyArrayObject *L_nzvals = NULL, *L_colind = NULL, *L_rowptr = NULL;
+    /* compressed sparse column matrix U */
+    int U_N=0, U_nnz=0;
+    PyArrayObject *U_nzvals = NULL, *U_rowind = NULL, *U_colptr = NULL;
+    /* right hand side / solution */
+    PyObject *X_py=NULL;
+    /* whether the matrix is transposed ('T'), conjugate transposed ('H') or normal ('N') */
+    volatile int itrans = 'N';
+    volatile jmp_buf* jmpbuf_ptr;
+    SLU_BEGIN_THREADS_DEF;
+
+    static char* kwlist[] = {
+        "trans",
+        "L_N", "L_nnz", "L_nzvals", "L_colind", "L_rowptr",
+        "U_N", "U_nnz", "U_nzvals", "U_rowind", "U_colptr",
+        "B", NULL
+    };
+
+    /* Parse and check input arguments. */
+
+    int res = PyArg_ParseTupleAndKeywords(args, keywds, "CiiO!O!O!iiO!O!O!O", kwlist,
+        &itrans,
+        &L_N, &L_nnz, &PyArray_Type, &L_nzvals, &PyArray_Type, &L_colind, &PyArray_Type, &L_rowptr,
+        &U_N, &U_nnz, &PyArray_Type, &U_nzvals, &PyArray_Type, &U_rowind, &PyArray_Type, &U_colptr,
+        &X_py );
+    if(!res)
+        return NULL;
+
+    volatile trans_t trans;
+    if (itrans == 'n' || itrans == 'N')
+        trans = NOTRANS;
+    else if (itrans == 't' || itrans == 'T')
+        trans = TRANS;
+    else if (itrans == 'h' || itrans == 'H')
+        trans = CONJ;
```

Comment:
Here you don't put single statements in blocks but elsewhere you do. For consistency could you do that here

Reply:
fixed in [f530ccf](https://github.com/scipy/scipy/pull/17924/commits/f530ccf711242e1aeaae2af257cecf0e8ff5037e)